2022-10-24 14:19:20,204 INFO    MainThread:38768 [wandb_setup.py:_flush():71] setting env: {}
2022-10-24 14:19:20,204 INFO    MainThread:38768 [wandb_setup.py:_flush():71] setting login settings: {}
2022-10-24 14:19:20,204 INFO    MainThread:38768 [wandb_init.py:_log_setup():371] Logging user logs to C:\Users\albu\Documents\GitHub\deep-multimodal-glioma-prognosis\wandb\run-20221024_141920-23xkm3pf\logs\debug.log
2022-10-24 14:19:20,204 INFO    MainThread:38768 [wandb_init.py:_log_setup():372] Logging internal logs to C:\Users\albu\Documents\GitHub\deep-multimodal-glioma-prognosis\wandb\run-20221024_141920-23xkm3pf\logs\debug-internal.log
2022-10-24 14:19:20,204 INFO    MainThread:38768 [wandb_init.py:_jupyter_setup():321] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x000002B463EDC970>
2022-10-24 14:19:20,207 INFO    MainThread:38768 [wandb_init.py:init():404] calling init triggers
2022-10-24 14:19:20,207 INFO    MainThread:38768 [wandb_init.py:init():409] wandb.init called with sweep_config: {}
config: {}
2022-10-24 14:19:20,207 INFO    MainThread:38768 [wandb_init.py:init():460] starting backend
2022-10-24 14:19:20,207 INFO    MainThread:38768 [backend.py:_multiprocessing_setup():99] multiprocessing start_methods=spawn, using: spawn
2022-10-24 14:19:20,211 INFO    MainThread:38768 [backend.py:ensure_launched():216] starting backend process...
2022-10-24 14:19:20,305 INFO    MainThread:38768 [backend.py:ensure_launched():221] started backend process with pid: 37396
2022-10-24 14:19:20,306 INFO    MainThread:38768 [wandb_init.py:init():469] backend started and connected
2022-10-24 14:19:20,307 INFO    MainThread:38768 [wandb_run.py:_label_probe_notebook():911] probe notebook
2022-10-24 14:19:20,308 INFO    MainThread:38768 [wandb_run.py:_label_probe_notebook():921] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2022-10-24 14:19:20,308 INFO    MainThread:38768 [wandb_init.py:init():533] updated telemetry
2022-10-24 14:19:20,366 INFO    MainThread:38768 [wandb_init.py:init():563] communicating current version
2022-10-24 14:19:22,375 INFO    MainThread:38768 [wandb_init.py:init():568] got version response upgrade_message: "wandb version 0.13.4 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2022-10-24 14:19:22,376 INFO    MainThread:38768 [wandb_init.py:init():578] communicating run to backend with 30 second timeout
2022-10-24 14:19:22,727 INFO    MainThread:38768 [wandb_init.py:init():606] starting run threads in backend
2022-10-24 14:19:26,042 INFO    MainThread:38768 [wandb_run.py:_console_start():1810] atexit reg
2022-10-24 14:19:26,042 INFO    MainThread:38768 [wandb_run.py:_redirect():1684] redirect: SettingsConsole.WRAP
2022-10-24 14:19:26,042 INFO    MainThread:38768 [wandb_run.py:_redirect():1721] Wrapping output streams.
2022-10-24 14:19:26,044 INFO    MainThread:38768 [wandb_run.py:_redirect():1745] Redirects installed.
2022-10-24 14:19:26,044 INFO    MainThread:38768 [wandb_init.py:init():633] run started, returning control to user process
2022-10-24 14:20:29,743 INFO    MainThread:38768 [wandb_init.py:_pause_backend():284] pausing backend
2022-10-24 14:20:29,744 INFO    MainThread:38768 [jupyter.py:save_ipynb():374] not saving jupyter notebook
2022-10-24 14:31:42,539 INFO    MainThread:38768 [wandb_init.py:_resume_backend():293] resuming backend
2022-10-24 14:31:42,642 INFO    MainThread:38768 [wandb_setup.py:_flush():71] setting env: {}
2022-10-24 14:31:42,642 INFO    MainThread:38768 [wandb_setup.py:_flush():71] setting login settings: {}
2022-10-24 14:31:42,642 INFO    MainThread:38768 [wandb_init.py:_log_setup():371] Logging user logs to C:\Users\albu\Documents\GitHub\deep-multimodal-glioma-prognosis\wandb\run-20221024_143142-2wi823jc\logs\debug.log
2022-10-24 14:31:42,642 INFO    MainThread:38768 [wandb_init.py:_log_setup():372] Logging internal logs to C:\Users\albu\Documents\GitHub\deep-multimodal-glioma-prognosis\wandb\run-20221024_143142-2wi823jc\logs\debug-internal.log
2022-10-24 14:31:42,643 INFO    MainThread:38768 [wandb_init.py:init():404] calling init triggers
2022-10-24 14:31:42,643 INFO    MainThread:38768 [wandb_init.py:init():409] wandb.init called with sweep_config: {}
config: {}
2022-10-24 14:31:42,643 INFO    MainThread:38768 [wandb_init.py:init():424] re-initializing run, found existing run on stack: 23xkm3pf
2022-10-24 14:31:42,643 INFO    MainThread:38768 [wandb_run.py:finish():1457] finishing run tomealbuquerque/deep-multimodal-glioma-prognosis/23xkm3pf
2022-10-24 14:31:42,644 INFO    MainThread:38768 [jupyter.py:save_history():429] not saving jupyter history
2022-10-24 14:31:42,644 INFO    MainThread:38768 [jupyter.py:save_ipynb():374] not saving jupyter notebook
2022-10-24 14:31:42,644 INFO    MainThread:38768 [wandb_init.py:_jupyter_teardown():303] cleaning up jupyter logic
2022-10-24 14:31:42,644 INFO    MainThread:38768 [wandb_run.py:_atexit_cleanup():1780] got exitcode: 0
2022-10-24 14:31:42,645 INFO    MainThread:38768 [wandb_run.py:_restore():1752] restore
2022-10-24 14:31:43,314 INFO    MainThread:38768 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 900
  total_bytes: 900
}

2022-10-24 14:31:43,477 INFO    MainThread:38768 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 900
  total_bytes: 900
}

2022-10-24 14:31:43,788 INFO    MainThread:38768 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 900
  total_bytes: 900
}

2022-10-24 14:31:43,923 INFO    MainThread:38768 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 900
  total_bytes: 8460
}

2022-10-24 14:31:44,039 INFO    MainThread:38768 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 900
  total_bytes: 8460
}

2022-10-24 14:31:44,147 INFO    MainThread:38768 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 900
  total_bytes: 8460
}

2022-10-24 14:31:44,256 INFO    MainThread:38768 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 900
  total_bytes: 8460
}

2022-10-24 14:31:44,364 INFO    MainThread:38768 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 8460
  total_bytes: 8460
}

2022-10-24 14:31:44,473 INFO    MainThread:38768 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 8460
  total_bytes: 8460
}

2022-10-24 14:31:44,582 INFO    MainThread:38768 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 8460
  total_bytes: 8460
}

2022-10-24 14:31:44,690 INFO    MainThread:38768 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 8460
  total_bytes: 8460
}

2022-10-24 14:31:44,799 INFO    MainThread:38768 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 8460
  total_bytes: 8460
}

2022-10-24 14:31:45,356 INFO    MainThread:38768 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 8460
  total_bytes: 8460
}

2022-10-24 14:31:45,898 INFO    MainThread:38768 [wandb_run.py:_wait_for_finish():1912] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 8460
  total_bytes: 8460
}
local_info {
}

2022-10-24 14:31:47,537 INFO    MainThread:38768 [wandb_run.py:_append_history():2130] rendering history
2022-10-24 14:31:47,537 INFO    MainThread:38768 [wandb_run.py:_append_summary():2085] rendering summary
2022-10-24 14:31:47,538 INFO    MainThread:38768 [wandb_run.py:_append_files():2180] logging synced files
